cluster-provision/gocli/opts/nfscsi/manifests/nfs-server.yaml

```diff
@@ -31,12 +31,14 @@ spec:
         cdi.kubevirt.io/testing: ""
     spec:
       containers:
-      - image: quay.io/kubevirtci/nfs-server-alpine:12
+      - image: docker.io/gists/nfs-server:2.6.4
```

Comment:
dockerhub is a pain, since it will start rate limiting us at some point

Reply:
Done